src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodVolumes/PvcVolume/config.jelly

```diff
@@ -0,0 +1,45 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-2009, Sun Microsystems, Inc.
```

Comment:
Well, its a copy and paste from the other config.jelly found under PodVolumes. :-)